config.js

```diff
@@ -1153,12 +1165,8 @@ function load_nsfs_nc_config() {
         const node_config = config_data.host_customization?.[node_name];
         const merged_config = _.merge(shared_config, node_config || {});
 
+        set_nc_nsfs_config_to_env(merged_config);
         Object.keys(merged_config).forEach(function(key) {
-            const config_to_env = ['NOOBAA_LOG_LEVEL', 'UV_THREADPOOL_SIZE', 'GPFS_DL_PATH'];
-            if (config_to_env.includes(key)) {
```

Comment:
when taking it out of this loop, you iterate the merged_config twice, is there a real reason to do so?